bionic/cache_api.py

```diff
@@ -115,34 +115,45 @@ def delete(self):
         """
         Safely deletes the artifact and its metadata from the cache.
 
-        Returns True if the entry is deleted and False if it was not found. Throws a
+        Returns True if the artifact was deleted and False if it was not found. Throws a
```

Comment:
Two booleans feels a bit clunky; and it would technically be a breaking change. TBH I doubt most users will even care about the first boolean, but I tried to pick the value they'd most likely care about. I don't see much reason for users to care about whether metadata files are deleted or not, especially since they can now automatically disappear.